inputFiles/lagrangianContactMechanics/contactMechanics.ats

```diff
@@ -45,6 +45,14 @@ decks = [
         partitions=((1, 1, 1), ),
         restart_step=1,
         check_step=2,
+        restartcheck_params=RestartcheckParameters(**restartcheck_params)),
+    TestDeck(
+        name="ContactMechanics_PEBICrack_smoke",
+        description=
+        "Two cubes with a fracture separating them (unstructured PEBI grid)",
+        partitions=((1, 1, 1), (2, 1, 1)),
+        restart_step=0,
+        check_step=10,
```

Comment:
Same comment as in  `poromechanicsFractures.ats`. Is the restart actually checked?

Reply:
like this it is not.